runtime/gc_vlhgc/IncrementalGenerationalGC.cpp

```diff
@@ -1928,6 +1946,13 @@ MM_IncrementalGenerationalGC::preConcurrentInitializeStatsAndReport(MM_Environme
 	stats->_scanTargetInBytes = _globalMarkPhaseIncrementBytesStillToScan;
 	env->_cycleState = &_persistentGlobalMarkPhaseState;
 	static_cast<MM_CycleStateVLHGC*>(env->_cycleState)->_vlhgcIncrementStats._markStats._startTime = j9time_hires_clock();
```

Comment:
existing code, but we should store TS into a local and use it both to store into the stats and for the hook
same for post process method

Reply:
Agreed, will refactor this portion.